Dockerfile

```diff
@@ -20,7 +20,7 @@ COPY . /adam/src
 ARG GOOS=linux
 ARG GOARCH=amd64
 
-RUN go build -o /adam/bin/adam main.go
+RUN go build -ldflags "-s -w" -o /adam/bin/adam main.go
```

Comment:
Why this?

Reply:
They [remove debug info](https://golang.org/cmd/link) and reduce the size of image.

Reply:
They remove some, but [not everything](https://github.com/golang/go/issues/19226). Do we care? We aren't distributing the image in a production capacity. How much difference does it make in size, before and after?

Reply:
18 vs 13 MB for binary

Reply:
I really don't care about the 5MB in size; do we use adam anywhere (on embedded systems) where it really matters? I guess if it already is in here, we can leave it.

Reply:
Well, removed for now.